lib/Twig/Error/Loader.php

```diff
@@ -24,8 +24,18 @@
  */
 class Twig_Error_Loader extends Twig_Error
 {
-    public function __construct($message, $lineno = -1, $filename = null, Exception $previous = null)
+    private $guess = false;
+
+    public function __construct($message, $lineno = -1, $source = null, Exception $previous = null)
+    {
+        parent::__construct($message, false, null, $previous);
+        $this->guess = true;
```

Comment:
When is it called on-demand? As noted in the phpdoc of the class, I don't think there is a case where we can guess for such exceptions.